### ISSUE TITLE
Localization of igEditors

### ISSUE THREAD

**Issue by Marina-L-Stoyanova** [IgniteUI/ignite-ui#352]
There are some changes in the description of the options/methods/events of the editors that need to be localized.

Here are the commits with the changes (please review the history of the editors file and see if I missed some comit):
ef52a8cc96a20d25fe849a08c4e3cde501de016f
a8fbc790f4382d72a0eee72f0bedf4fe24fc98e0
6bb91d4967830b1adc6cdc6e6d270dbf9c62ba2c
dc7f8f4f769678227fbd8660c9897c71db80cc74
6324d014eb735114dcd511d46949df8c236499a6
4d039e77fd45ae10b57f2a484fcf1355c3032895
434443c4b1705c0d3b7f802ca7696e8dd0344ae6
ca7d2cf2a59e60e8ba3eb7e22305f06676c03665

**Comment by IvayloG** [comment]
Several method descriptions to be localized.
Here are the commits with the changes:
c243e5b3a425fcf835ac99794ea3aed69c4d21f7

**Comment by ig-ealbert** [comment]
Due to the volume of changes as well as how many .xml files these changes impact, I might not be able to complete them for 16.2.  We'll see.

**Comment by ig-ealbert** [comment]
I've made some changes so far in TFS but it's really really really hard to follow the changes in the source code and figure out how they apply to the different editor .xml files in TFS.  I think I may be causing more harm than good here.  Any suggestions?   I can try the file diff on the xmls but that's also a huge mess and it shows almost EVERYTHING as changed.  I might just not make these changes, as I don't see any safe or efficient way to do it.

**Comment by ig-ealbert** [comment]
I've done the best I can with this in TFS.  Please close the issue.
